### DISCUSSION TITLE
Mail about the useage of codespaces spenidng limit, but no active codespaces are found.

### DISCUSSION THREAD

**Discussion by sethsnel** [community/community#59722] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Lately I've received this mail from GH:
![image](https://github.com/community/community/assets/3225809/c2aba3c6-ae8d-486c-9040-f76df087ec7e)

When I navigate to `https://github.com/codespaces`, I get to see exactly this screen:
![image](https://github.com/community/community/assets/3225809/96d7b182-d587-45f3-b898-68f069223175)

As far as I can see I have no active codespaces. So what is incurring/occupying my free storage useage?

**Comment by Fengyi-Li** [comment, marked as answer]
Quote this reply in another thread, did you try to check your usage report? https://github.com/orgs/community/discussions/59588#discussioncomment-6309195

**Reply by sethsnel** [reply]
Thanks, it was indeed some long forgotten prebuild for one of my repo's eating up the space.

**Comment by robespierre81** [comment]
I have the same problem. Here I checked the link, but I cannot create any usage report. Only an overview can be seen. I have no codespaces and It is not possible to see, where are so called prebuilds
